name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/Extractor.java

```diff
@@ -595,11 +595,15 @@ else if (this instanceof IBFlexStatementExtractor)
                         .collect(Collectors.toList());
 
         Map<Extractor, List<Item>> itemsByExtractor = new HashMap<>();
-        itemsByExtractor.put(this, result);
+        itemsByExtractor.put(this, removeDuplicateDividendsWithoutTax(result));
```

Comment:
Zwei Anmerkungen:
Erstens wird diese Methode **nur** von den Tests aufgerufen. Der Hintergrund steht in dem Kommentar oben in der Methode: irgendwann habe ich da was ändern müssen, und wollte vermeiden hunderte von Tests anpassen zu müssen. Der Aufruf muss also mindestens noch in die #extract Methode rein - die wird aber erst in Subklassen implementiert.

Zweitens sollten wir den Namen etwas allgemeiner halten. Vielleicht #postprocessing()?

Reply:
Danke für die Anmerkungen - das der Extractor nur in Tests verwendet wird, habe ich nicht bedacht.
Um alle inputFiles vergleichen zu können, muss ich dann etwas weiter nach oben
- In AbstractPDFExtractor.#extract gibt es nur ein File
- PDFImportAssistant.#run hat alle inputFiles

Ich müsste also in `#run` nach der `itemsByExtractor`-Sammlung über alle Extractor `#postprocessing` aufrufen - Versuche ich heute abend mal.

Dein Namensvorschlag ist auf jeden Fall besser.